indico/modules/events/management/controllers.py

```diff
@@ -178,3 +178,13 @@ def _update_session_coordinator_privs(self, form):
             log_msg = 'Session coordinator privilege changed to {}: {}'.format(form.data[priv_field],
                                                                                COORDINATOR_PRIV_TITLES[priv_field])
             self.event_new.log(EventLogRealm.management, EventLogKind.positive, 'Protection', log_msg, session.user)
+
+
+class RHMoveEvent(RHConferenceModifBase):
```

Comment:
My reasoning was the same as @ThiefMaster's. Since we already have actions like `RHDeleteEvent` or `RHLockEvent` I thought that we should put the `RHMoveEvent` in the same file.